### ISSUE TITLE
[Accessibility][PDF Viewer] 3 dot icon menu uses ARIA attributes improperly

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1734]
**Steps to Reproduce**
1. Enable NVDA.
2. Open the page and navigate to three dots icon button. Note that it is announced as a menu button.
3. Activate the button to expand the related dialog. Note that the expanded state is not announced.
4. Dismiss the related dialog by pressing the Esc key. Note that NVDA announces the button as expanded, although the related dialog is now closed.

**Current Behavior**
The 3 dot icon menu button adds and removes the aria-expanded attribute rather than changing the value. As a result, screen reader users are not correctly informed of the expanded or collapsed state of the related content.

Advisory: The aria-haspopup attribute is used unnecessarily. With a value of "true" for the aria-haspopup attribute, the button may be interpreted by screen readers as a menu button, leading screen reader users to expect certain related roles and functionality that are not implemented here. NVDA announces this button as "menu button, submenu".

**Expected behavior/code**
The aria-expanded attribute is present at all times, but the value changes appropriately ("true" when dialog is expanded, "false" otherwise).

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users.

**WCAG SC**
4.1.2 Name, Role, Value

**Possible Solution**
Ensure the aria-expanded attribute is present at all times, but the value changes appropriately ("true" when dialog is expanded, "false" otherwise). 

Consider removing the aria-haspopup attribute. Consider following the WAI-ARIA Authoring Practices design pattern for Disclosure, to simplify this experience:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-navigation.html

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
